core/controller/src/main/scala/whisk/core/loadBalancer/InvokerSupervision.scala

```diff
@@ -86,6 +126,21 @@ class InvokerPool(
 
         case GetStatus => sender() ! invokerStatus.toMap
 
+        case msg: InvocationFinishedMessage => {
+            // Forward message to invoker, if InvokerActor exists
+            invokers.get(msg.name).map(_.forward(msg))
+        }
+
+        case msg: ActivationRequest => {
```

Comment:
all activation requests are now serialized through this actor - do we really want to do this? since this actor processes different messages, including health pings, it seems like we can end up delaying a producer send unnecessarily. this changes the role of the actor from a supervisor to a gatekeeper for all invoker operations.